lib/conversations_client.ex

```diff
@@ -69,6 +69,13 @@ defmodule ExMicrosoftBot.Client.Conversations do
     |> change_deserialized_response_to_ok
   end
 
+  @doc """
+  This method allows you to update an activity. [API Reference](https://docs.microsoft.com/en-us/azure/bot-service/rest-api/bot-framework-rest-connector-api-reference?view=azure-bot-service-4.0#update-activity)
+  """
+  @spec update_activity(String.t, String.t, Models.Activity.t) :: :ok | Client.error_type
+  def update_activity(conversation_id, activity_id, %Models.Activity{service_url} = activity), do: update_activity(service_url, conversation_id, activity_id, activity)
```

Comment:
```
%Models.Activity{serviceUrl: service_url}
```
and group arity as in `reply_to_activity` (arity 3 then arity 4)

Reply:
Done